terminal_api_flows/flows/credit_txn.py

```diff
@@ -106,7 +107,17 @@ def credit_to_cash_transaction(total=0, skip_tip=False, skip_reward_notification
 
 
 @terminal_ping_decorator_3_attempts
-def credit_transaction(total=0, skip_tip=False, skip_reward_notification=False, skip_signin=False, allow_discount=True):
+def credit_transaction(
+        total=0,
+        skip_tip=False,
+        skip_reward_notification=False,
+        skip_signin=False,
+        allow_discount=True,
+        # Skip screens in real time if they come up
+        skip_tips_screen=False,
+        skip_reward_notification_screen=False,
+        skip_approval_screen=False
```

Comment:
Does this mean that, for this client, you still need to know which screens to skip before calling `credit_transaction`?

Reply:
Yeah this client works differently from the other two. Its more a "flow" client so you need to know ahead of time. The other two will utilize this new feature in much more "real time".